card/mod/standard/set/all/rich_html/form.rb

```diff
@@ -92,21 +92,21 @@ def inline_nests_editor?
 
   # test: are we opening a new multi-card form?
   def multi_card_editor?
-    nests_editor? ||                         # editor configured in voo
-      voo.structure || voo.edit_structure || # structure configured in voo
-      card.structure ||                      # structure in card rule
-      edit_fields.present?                   # list of fields in card rule
+    voo.structure || voo.edit_structure || # structure configured in voo
+      card.structure ||                    # structure in card rule
+      edit_fields?                         # list of fields in card rule
+  end
+
+  # override and return true to optimize
+  def edit_fields?
```

Comment:
(this simplification was driven by the wikirate value handling, which was the only place using `voo.editor = :nests`)